### PR TITLE
Update page in Connect API Manager “tips-troubleshooting-and-limitations”

### PR DESCRIPTION
**Pull request by dfeldick**
Automatically generated by Netlify CMS

### CODE REVIEW

**Comment by dfeldick** [comment]
@sbolosan @jcollins-axway @vivekschauhan @kflan-io please review

**Comment by dfeldick** [comment]
new error codes for https://jira.axway.com/browse/APIGOV-15844

**Review by jcollins-axway — APPROVED** [review]

**Review by kflan-io — APPROVED** [review]

File: content/en/docs/central/connect-api-manager/tips-troubleshooting-and-limitations.md [hunk 1]
```diff
@@ -79,17 +79,22 @@ A change of front-end image only is not detected. Changing the name of the API w
...
-| 3005 | could not make request to API Manager                                                                         | See [Administer API Manager network traffic](https://docs.axway.com/bundle/axway-open-docs/page/docs/central/connect-api-manager/network-traffic-apimanager/index.html) and [Agent variables](https://docs.axway.com/bundle/axway-open-docs/page/docs/central/connect-api-manager/agent-variables/index.html) |
+| 3005 | could not make request to API Manager                                                                        | See [Administer API Manager network traffic](https://docs.axway.com/bundle/axway-open-docs/page/docs/central/connect-api-manager/network-traffic-apimanager/index.html) and [Agent variables](https://docs.axway.com/bundle/axway-open-docs/page/docs/central/connect-api-manager/agent-variables/index.html)      |
+| 3006 | could not create the body of the request to API Manager | Internal system error. No user resolution |
```
Comment (sbolosan) [inline content/en/docs/central/connect-api-manager/tips-troubleshooting-and-limitations.md:85]: I get it... but it just seems like "so what?". "What now?"...

Comment (dfeldick) [reply content/en/docs/central/connect-api-manager/tips-troubleshooting-and-limitations.md:85]: Contact the system admin?

**Review by vivekschauhan — APPROVED** [review]

File: content/en/docs/central/connect-api-manager/tips-troubleshooting-and-limitations.md (hunk 1 above)
Comment (sbolosan) [reply content/en/docs/central/connect-api-manager/tips-troubleshooting-and-limitations.md:85]: discussed... resolved...

Comment (dfeldick) [reply content/en/docs/central/connect-api-manager/tips-troubleshooting-and-limitations.md]: I changed it from "no user resolution" to "contact your system administrator"

**Review by lbadenhop — APPROVED** [review]

**Pull request merged**